test/e2e/denylist_test.go

```diff
@@ -20,8 +20,8 @@ import (
 
 	"github.com/gogo/protobuf/proto"
 
-	testFramework "github.com/prometheus-operator/prometheus-operator/test/framework"
 	"github.com/pkg/errors"
+	testFramework "github.com/prometheus-operator/prometheus-operator/test/framework"
```

Comment:
In this test we seem to only test ServiceMonitor and Prometheus custom resources, would be good to also add ThanosRuler test, what do you think? I believe the test was written before the addition of that CR, so would be nice since we are changing it to test that resource as well. Does that make sense?

Reply:
definitely, let me add that e2e test.